src/parser.ts

```diff
@@ -3621,7 +3621,8 @@ export function parseMemberOrUpdateExpression(
   inGroup: 0 | 1,
   start: number,
   line: number,
-  column: number
+  column: number,
+  chained: 0 | 1 = 0
```

Comment:
@aladdin-add `chained: 0 | 1 = 0` is temporary because we don't know the ESTree specs yet. Added to in line with Babel parser :)  When you figure it out with ESTree this has to either 1) be removed or 2) moved up to be in sync with`inGroup` and `= 0` removed so every call to `parseMemberOrUpdateExpression` has to have a `0` as func arg.

Example: `parseMemberOrUpdateExpression(parser, context, /* inGroup */ 0, /* chained */ 0, start, line, column)`

If you figure out that the `OptionalMemberExpression` shouldn't be there, well just remove it and use the original one and update `estree.ts` as well.